### ISSUE TITLE
TCGAanalyze_DEA explanation

### ISSUE THREAD

**Issue by Rahel14350** [BioinformaticsFMRP/TCGAbiolinks#293]
Dear @tiagochst,
I have a question regarding DEA analysis. In the workflow you explained this analysis will find the DEG between Cond1 and Cond2 but it is not clear for me which one is the base level. For example when I want to compare the Tumor samples vs Normal (base level), then which one is Cond1 (mat1)? 

```
 dataDEGs <- TCGAanalyze_DEA(mat1 = dataFilt[,samplesNT],
                                  mat2 = dataFilt[,samplesTP],
                                  Cond1type = "Normal",
                                  Cond2type = "Tumor")

```
Many thanks in advance,
Rahel

**Comment by torongs82** [comment]
Dear @Rahel14350 thank you for using our TCGAbiolinks. If you consider Normal as base level, in the function TCGAanalyze_DEA you can see sampleNT of matrix 1 (mat1) that are normal samples or condition 1 (Cond1), while samplesTP of matrix 2 (mat2) are tumor samples or condition 2 (Cond2). Did  this helped you? Thank you, Antonio.

**Comment by Rahel14350** [comment]
Dear @torongs82 
Many thanks for your prompt reply. Yes, this is what I wanted to make sure about it. For example when mat1 is Normal and mat2 is Cancer tissue, I can conclude that the upregulated DEGs are upregulated in cancer tissues and not in the normal one. Am I correct?

**Comment by torongs82** [comment]
Yes exactly, genes in dataDEGs that showed logFC > 1 and FDR < 0.01 for example, can be considered significantly Up-regulated genes in cancer (TP) tissues. I will add some boxplots in our vignette to show this differences. But you can also run this code to have an idea 
''' # DEGs table with expression values in normal and tumor samples
dataDEGsFiltLevel <- TCGAanalyze_LevelTab(dataDEGs,"Tumor","Normal",
                                          dataFilt[,samplesTP],dataFilt[,samplesNT])''' 
Best.

**Comment by Rahel14350** [comment]
Thank you so very much for your help!
